dev/integration_tests/flutter_gallery/test/demo/cupertino/cupertino_navigation_demo_test.dart

```diff
@@ -32,7 +32,7 @@ void main() {
     // Tap some row to go to the next page.
     await tester.tap(find.text('Buy this cool color').first);
     await tester.pump();
-    await tester.pump(const Duration(milliseconds: 500));
+    await tester.pump(const Duration(milliseconds: 600));
```

Comment:
Is this based on CupertinoRouteTransitionMixin.transitionDuration?  I wonder if you could base it on that.  No big deal if not, though.